### ISSUE TITLE
[Q] Is data on the Y axis overwritten (with the same X axis value) when restarting the training from a checkpoint?

### ISSUE THREAD

**Issue by hkproj** [wandb/wandb#6728]
Hello!

I have a training script that uses a custom X axis (called "global_step") and defined as follows:

```python
# define our custom x axis metric
wandb.define_metric("global_step")
# define which metrics will be plotted against it
wandb.define_metric("validation/*", step_metric="global_step")
wandb.define_metric("train/*", step_metric="global_step")
```

I log the training loss as follows:

```python
# Log the loss
wandb.log({'train/loss': loss.item(), 'global_step': global_step})
```

Imagine I train the model for `global_step` 0, 1, 2, ... 1000 but then I restart the training (due to a crash) from the `global_step` 500
Will the data from `global_step` 500 onward be overwritten (as I expect)? Looking at the charts I couldn't see if the data was being overwritten or not.

Thank you!

**Comment by Carlo-Argel** [comment]
Hi @hkproj ,

Thank you for reaching out. Wandb step must be monotonically increasing in each call, otherwise the step value is ignored during your call to log(). Currently wandb does not support overwriting historical steps. Our recommendation in this case is to create a new run and log new data in it for the continued training. You can additionally add a group name to these runs to keep track of them in your workspace as being related to each other. We will log a feature request for this, please let me know how this impacts your workflow and why this is important to you

**Comment by hkproj** [comment]
Thanks @Carlo-Argel I'll try to create a group. 

I think restarting after a crash/forced-stop is such a common operation in machine learning that I think it should be handled out of the box, for example W&B could add a parameter to the `log` function to indicate if the value should replace old historical records that have the same step (value of the X axis).
